docs/enhancement-proposals/off-ep-0004.md

```diff
@@ -0,0 +1,100 @@
+# OFF-EP 0004 — Handlers specify their dependence on other handlers
+
+**Status:** Proposed
+
+**Authors:** Matt Thompson
+
+**Stakeholders:** David Mobley, John Chodera, Jeffrey Wagner, Simon Boothroyd
+
+**Acceptance criteria:** <Unanimity>
+
+**Created:** 2021-12-01
+
+**Discussion:** [Issue #2](https://github.com/openforcefield/standards/issues/2)
+
+**Implementation:** [PR #24](https://github.com/openforcefield/standards/pull/24)
+
+## Abstract
+
+This OFF-EP requires that handlers state which other handlers they depend on, if any.
+
+## Motivation and Scope
+
+There are some cases in which it is not clear which parameter section(s) in a SMIRNOFF force field
+depend on each other. For example, it is natural to define a TIP3P force field with a
+`<LibraryCharges>` section but neglect to add an `<Electrostatics>` section. This leads to
+difficulties and strange behaviors in implementation. For example, there is no way to know what
+cutoff method and distance should be used when the `<Electrostatics>` section is missing. This
+ambiguity could be resolved by requiring that, if a force field defines any charge methods, it must
+have a `<Electrostatics>` section. This sort of approach should be applied generally to all
+handlers; developers implementing this SMIRNOFF specification may intuit these relationships over
+time, but a better solution would be clarifying them in the specification itself.
+
+This appears to only be an issue when handlers **require** information from other handlers. There
+are some cases in which one section **could** use information from another but does not **require**
+it. For example, a `<Bonds>` section does not **require** to information from other sections,
```

Comment:
"For example, a `<Bonds>` section does not **require** to information from other sections,"

This sentence contains grammatical issues. I'm not sure what is meant.